getting-started-spring-boot-maven/jqassistant/maven.xml

```diff
@@ -0,0 +1,29 @@
+<jqassistant-rules
+        xmlns="http://schema.jqassistant.org/rule/v2.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://schema.jqassistant.org/rule/v2.0 https://schema.jqassistant.org/rule/jqassistant-rule-v2.0.xsd">
+
+    <!-- tag::content[] -->
+    <!--1-->
+    <group id="maven:Default">
+        <includeConcept refId="maven:*"/>
+    </group>
+
+    <!--2-->
+    <concept id="maven:ProjectArtifactDependencies">
```

Comment:
This is a concept just for reporting purposes, it should have an according suffix in its id, e.g. "maven:ProjectArtifactDependencyReport"